pkg/cmd/pipelinerun/cancel.go

```diff
@@ -75,7 +76,7 @@ func cancelPipelineRun(p cli.Params, s *cli.Stream, prName string) error {
 	}
 
 	prCond := formatted.Condition(pr.Status.Conditions)
```

Comment:
Is not better to don't format the condition here and do the decision bases on Condition Status, if the Condition Status is True or False then don't do the cancellation, if the condition status in unknown then do the cancellation

cc @vdemeester @danielhelfand

Reply:
Yes, I think the original reason for using formatting here was a quick fix for the problem. Maybe what we can do here, since we are approaching the release, is keep this approach as is for now but open a separate issue to refactor based on status.

Reply:
@piyush-garg agreed. Depending on how easy/hard it is to modify this, I would go with @danielhelfand suggestion :upside_down_face:

Reply:
Yes, let's create a refactor issue for this and handle later @PuneetPunamiya